### PR TITLE
add ConfigurationJobConfig.java to load the context we need to run th…

### PR DESCRIPTION
**Pull request by iguannago**
implement Option2: use config classes to load the context needed to run the jobs

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -227,9 +227,9 @@ dependencyManagement {
 }
 
 repositories {
-  mavenLocal()
-  maven { url 'https://jitpack.io' }
   mavenCentral()
+  maven { url 'https://jitpack.io' }
+  jcenter()
```
Comment (indvamshi) [inline build.gradle:232]: jcenter is deprecated can't use anymore.

File: src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/job/ConfigurationJobConfig.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "job.configuration")
+@Getter
+@Setter
+public class ConfigurationJobConfig {
+    String camundaMaxResults;
+}
```
Comment (indvamshi) [inline src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/job/ConfigurationJobConfig.java:14]: Do we really need a pojo to use this property, can't it be used as @Value

**Review by indvamshi — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (iguannago) [reply build.gradle:232]: fixed.

File: src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/job/ConfigurationJobConfig.java (hunk 2 above)
Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/wataskmonitor/config/job/ConfigurationJobConfig.java:14]: Yes, as you mentioned, we can use a @value, but I decided to use this approach to set a framework. We will most likely have to add different contexts/configurations to run different jobs, which seems cleaner than having a few different @values in different jobs. That's the main reason.

**Review by indvamshi — APPROVED** [review]

**Pull request merged**